runtime/cpp/emboss_text_util.h

```diff
@@ -69,7 +69,7 @@ class TextOutputOptions final {
 
   TextOutputOptions WithNumericBase(int new_value) const {
     TextOutputOptions result = *this;
-    result.numeric_base_ = new_value;
+    result.numeric_base_ = static_cast<::std::uint8_t>(new_value);
```

Comment:
Please add /**/ between < and : to avoid the "<:" digraph that is enabled on some compilers; i.e.:

static_cast</**/ ::std::unit8_t>

Reply:
Done